### PR TITLE
crit logging

### PR DESCRIPTION
**Pull request by toomykins**
basic proc that logs when you go into crit

### CODE REVIEW

**Comment by Eneocho** [comment]
Does this also log when you get out of crit?

**Comment by toomykins** [comment]
no

File: code/modules/mob/living/damage_procs.dm [hunk 1]
```diff
@@ -52,6 +53,10 @@
 	return 1
 
 
+/mob/living/proc/critlog(curH,prevH) //current health, previous health
+	if(stat == UNCONSCIOUS && curH < 0 && curH > -95.0 && prevH > 0)
+		src.attack_log += text("\[[time_stamp()]\] <font color='red'>Has gone into CRIT!</font>")
```
Comment (ShiftyRail) [inline code/modules/mob/living/damage_procs.dm]: remove `text`

**Comment by ghost** [comment]
i dont see the point of this most of the time

**Comment by Eneocho** [comment]
> 
> 
> i dont see the point of this most of the time

More detailed logs are always helpful, if nothing else.

**Pull request merged**